code/datums/stars.dm

```diff
@@ -97,7 +97,7 @@
 
```

Comment:
More likely: if(!main_dock) is somehow causing this, move this down to the bottom so it checks FOB_SHUTTLE first. if RELAY is removed, it would just stop at FOB_SHUTTLE anyways